query/view/pop_subquery.js

```diff
@@ -0,0 +1,16 @@
+
+var peliasQuery = require('pelias-query');
+
+/**
+  Population / Popularity subquery
+**/
+
+module.exports = function( vs ){
```

Comment:
could this possibly go into the pelias-query module? seems like the type of thing we typically keep in there.